CMakeLists.txt

```
@@ -118,6 +118,29 @@ else()
     endif()
 endif()
 
+# Set the SO version. The SO version must be incremented every time a change
+# occurs to the ABI that causes a backward incompatibility. Such changes
+# include, exhaustively:
+#
+# * a change to struct or class layout
+# * enum changes that would cause a renumbering of previously published enums
+# * a removal of a struct, class, enumeration, or function
+# * a change in parameters to a free standing function
+# * a removal of a global variable
+#
+# OTIO currently designates the minor version number for breaking changes,
+# e.g. v0.15, v0.16.0, v0.17.0, accordingly the SO version will be incremented
+# to match. So for example, the following OTIO versions would map to these
+# ABI versions:
+#
+# * v0.18.0 - 0.18
+# * v0.19.0 - 0.19
+# * v0.19.1 - 0.19 # No ABI changes with minor version changes
+# * v1.0.0 - 1.0
+# * v1.1.0 - 1.1
+#
+set(OTIO_SOVERSION ${OTIO_VERSION_MAJOR}.${OTIO_VERSION_MINOR})
```

Although you can set a dotted version for SOVERSION it will cause issues in  tooling that thinks SOVERSION is an integer. 

That's why I was suggesting SOVERSION should be major*10 + minor.

```suggestion
math(EXPR OTIO_SOVERSION "${OTIO_VERSION_MAJOR} * 100 + ${OTIO_VERSION_MINOR}")
```

---

Ah OK, that makes sense.